pmultiqc/main.py

```
@@ -74,7 +74,7 @@ def pmultiqc_plugin_execution_start():
             if os.path.exists(output_dir):
                 analysis_dir_new.append(output_dir)
             else:
-                raise SystemExit(f"Illegal file path: {output_dir}")
+                raise ValueError(f"Illegal file path: {output_dir}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)